pkg/obsoletetests/ocp_obsoletetests.go

```diff
@@ -20,6 +20,10 @@ var obsoleteTests = sets.New[obsoleteTestIdentifier](
 			name:  "[sig-arch] Check if alerts are firing during or after upgrade success",
 			suite: "Cluster upgrade",
 		},
+		{
+			name:  "OCP-12158:APIServer Specify ResourceQuota on project",
+			suite: "remote registry related scenarios",
+		},
```

Comment:
What's the background for marking this obsolete?

Reply:
Add the context to obsolete this test case in the PR.